### PR TITLE
fix telegram webhook

### LINKED ISSUE #29837: Telegram webhook broken
### Description

For some reason, the telegram webhook isn't longer sent correctly (the HTML? containing the link is displayed raw). This should not be a Telegram issue as my own bot doing similar styled notifications works without issues.

Screenshot_20240316-101007

### Gitea Version

1.22.0+dev-1060-g6ead30dbc (latest nightly)

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

Rootful docker

### Database

SQLite

### PR DESCRIPTION
**Pull request by denyskon**
Fix #29837 which is a regression caused by https://github.com/go-gitea/gitea/pull/29145/files#diff-731445ee00f0f1bf2ff731f4f96ddcf51cdc53fd2faaf406eb3536fc292ea748L48. The line was probably removed by accident.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

File: services/webhook/telegram.go [hunk 1]
```diff
@@ -181,7 +181,8 @@ func (t telegramConvertor) Package(p *api.PackagePayload) (TelegramPayload, erro
 
 func createTelegramPayload(message string) TelegramPayload {
 	return TelegramPayload{
-		Message: strings.TrimSpace(message),
+		Message:   strings.TrimSpace(message),
+		ParseMode: "HTML",
```
Comment (wxiaoguang) [inline services/webhook/telegram.go]: I guess it is also good to set `DisableWebPreview = true`?

**Comment by wxiaoguang** [comment]
Could there be a new test in `TestTelegramPayload` to cover this case?

File: services/webhook/telegram.go (hunk 1 above)
Comment (denyskon) [reply services/webhook/telegram.go]: Done

**Comment by denyskon** [comment]
@wxiaoguang Added test

**Comment by wxiaoguang** [comment]
->  Refactor webhook #31587 
->  Use old behavior for telegram webhook #31588

**Pull request merged**